tests/test_excelparser/test_excelparser.py

```diff
@@ -143,5 +155,5 @@ def test_excelparser_only_classes(repo_dir: "Path") -> None:
         update_xlspath, force=True, input_ontology=ontology
     )
     assert updated_onto.ATotallyNewPattern
-    assert updated_onto.Pattern.iri == onto.Pattern.iri
+    assert updated_onto.FinitePattern.iri == onto.FinitePattern.iri
```

Comment:
We should check both Pattern (in the imported ontology) and FinitePattern (newly created)

Reply:
Pattern doesn't exists in 1.0.0-beta5, which is the one we have here.